_pages/projects.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Projects
+description: >-
+   The projects below are only a subset of the leading edge Arm developments actively being worked on by Linaro, its member partners, and the open source community.  Join us in the common goal of accelerating your product deployment within the Arm ecosystem!
```

Comment:
Should we perhaps have a cleaner description than copying the (old) opening text?